civihr_default_theme/template.php

```diff
@@ -406,6 +406,7 @@ function civihr_default_theme_preprocess_page(&$variables) {
   // Adds the theme path to Javascript variable.
   $path = drupal_get_path('theme', 'civihr_default_theme');
   drupal_add_js(array('civihr_default_theme' => array('path' => $path)), 'setting');
+  drupal_add_library('system', 'ui.datepicker');
```

Comment:
Did you eventually find out why this was not included for onboarding but needs to be explicitly included for this view?
Is there something that was done differently there?

Reply:
Yes, in onboarding we are using webforms which includes datepicker in `sites/all/modules/contrib/webform/components/date.inc`